libkbfs/folder_block_manager.go

```diff
@@ -209,6 +209,13 @@ func (fbm *folderBlockManager) enqueueBlocksToDelete(toDelete blocksToDelete) {
 	fbm.blocksToDeleteChan <- toDelete
 }
 
+func (fbm *folderBlockManager) enqueueBlocksToDeleteAfterShortDelay(
+	toDelete blocksToDelete) {
+	fbm.blocksToDeleteWaitGroup.Add(1)
+	time.AfterFunc(10*time.Millisecond,
```

Comment:
Yeah sorry, I merged it without review because I was seeing a lot of log spam and CPU usage last night due to a near-infinite receive-send loop.  Probably would be better as a const, I'll clean it up one day.